examples_src/vector-esri-edit.html

```diff
@@ -0,0 +1,20 @@
+---
+template: example.html
+title: esri ArcGIS REST Feature Service example with editing
+shortdesc: Example of using an ArcGIS REST Feature Service in an editing application.
+docs: >
+  This example loads features from ArcGIS REST Feature Service and allows to add new features or update existing features.
+tags: "vector, esri, ArcGIS, REST, Feature, Service, bbox, loading, server, edit, updateFeature, addFeature"
+---
+<div class="row-fluid">
+  <div class="span12">
+    <div id="map" class="map"></div>
+    <form class="form-inline">
+      <label>Draw or modify &nbsp;</label>
+      <select id="type">
+        <option value="DRAW">Draw</option>
+        <option value="MODIFY">Modify</option>
+      </select>
+    </form>
+  </div>
+</div>
```

Comment:
When you rebase on master, you'll want to put these examples in `examples` instead of `examples_src`.